src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ReadFile;
+import core.basesyntax.dao.ReadFileImpl;
+import core.basesyntax.process.DataProcess;
+import core.basesyntax.report.ReportCreator;
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.FruitTransaction;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import core.basesyntax.writereport.ReportWriter;
+import core.basesyntax.writereport.WriteReportImpl;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Map<FruitTransaction.Operation, OperationHandler> strategyMap = Map.of(
+                FruitTransaction.Operation.BALANCE, new BalanceHandler(),
+                FruitTransaction.Operation.RETURN, new ReturnHandler(),
+                FruitTransaction.Operation.SUPPLY, new SupplyHandler(),
+                FruitTransaction.Operation.PURCHASE, new PurchaseHandler()
+        );
+        String readFilePath = "src/main/resources/file.txt";
+        String reportFilePath = "src/main/resources/report.txt";
+        FruitStorage fruitStorage = new FruitStorage();
+        ReadFile readFileDao = new ReadFileImpl();
+        DataProcess dataProcess = new DataProcess();
+        dataProcess.addDataToStorage(readFileDao.readFromFile(readFilePath),
+                fruitStorage, strategyMap);
```

Comment:
see main() method structure.
Each block must be in a separate code line 
I do not see where do you convert file lines to java objects (`List<FruitTransaction>`)
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/e4dce91b-6240-417c-88ab-4c8a6e2112c0)